x/hub-genesis/types/state.go

```diff
@@ -1,8 +1,21 @@
 package types
 
 import (
+	"strings"
+
 	errorsmod "cosmossdk.io/errors"
 	host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
+	gerrc "github.com/dymensionxyz/gerr-cosmos/gerrc"
+)
+
+// ChannelState represents the state of a channel in the genesis bridge process
+type ChannelState bool
+
+const (
+	// WaitingForAck indicates the channel is waiting for acknowledgment
+	WaitingForAck ChannelState = false
+	// NeedsRetry indicates the channel needs to retry the genesis bridge
+	NeedsRetry ChannelState = true
 )
```

Comment:
why do we need it? 
seems like all we care is if there is a pending channel in the store or not..

Reply:
u don't want to retry on channels which are not acked yet